src/views/IntroScreen.html

```diff
@@ -1,28 +1,24 @@
-<div class="header">
-    <div class="pull-left w2 ml10">
-        <img src="resources/img/greenmarket_logo.svg">
-    </div>
-    <h1 data-bind="click: $screen.onLogin" class="pull-right mr20 mt15" style="text-decoration: underline;">Login</h1>
-</div>
-<div class="body">
-    <div class="row mt50 mb25 flexCenterCol">
-        <img src="resources/img/greenmarket_logo.svg" style="max-width:100px;">
-    </div>
-    <div class="row mb30" style="margin-top: 5px">
-        <div class="col-sm-4 col-md-6 offset-sm-4 offset-md-3">
-            <img src="resources/img/greenmarket.svg" style="width:100%">
-        </div>
-    </div>
-    <div class="row mb40" style="text-align:center">
-        <h1 style="font-size: 2em; font-weight: bold;">Online fresh-produce market place.</h1>
-    </div>
-    <div class="row">
-        <div class="col-md-4 col-lg-4 col-xl-2 offset-md-4 offset-lg-4 offset-xl-5">
-            <h1 class="pull-left" style="font-size: 45px">Buy</h1>
-            <h1 class="pull-right" style="font-size: 45px">Sell</h1>
+<body>
+
+    <header>
+        <a class="pull-right"><img id="imgAvatar" src="src/img/avatar.svg">Login</a>
+    </header>
+    
+    <div id="containerLanding" class="">
+        
+<<<<<<< HEAD
+        <img id= imgTitleIcon src="src/img/logoIcon.svg">
+=======
+        <img id= imgTitleLogo src="src/img/logoIcon.svg">
+>>>>>>> 406e25cf5fcf660f7af1e66a284ca29893fa7f3a
```

Comment:
Hierdie beteken jy het een branch try rebase op 'n ander branch en nie die merge conflicts resolve voor jy met die rebase continue het nie. Dan, die <img> tags in hierdie html file se src attributes wys almal na die verkeerde goed toe.
Die path moet wees 'resources/img/***.svg'. En die images wat jy in gecheck het se name is greenIcon en greenText en in die html soek jy vir logoIcon en logoText.
Dan haal asb die id's uit. Gee hulle klasse en gebruik waar jy kan die klasse in main.css. Amper alles wat ek daar in die less sien is klaar klasse wat jy in die html kan gee.